src/renderer/components/ModalQuizz/ModalQuizz.tsx

```diff
@@ -0,0 +1,221 @@
+  title,
+  steps,
+  isOpen,
+  onClose = () => {},
+  started,
+  StartScreen,
+  dismissable = true,
+}: Props) => {
+  const [stepIndex, setStepIndex] = useState(0);
+  const stepCount = steps.length;
+
+  const { t } = useTranslation();
+
+  const [userChoiceIndex, setUserChoiceIndex] = useState(undefined);
+
+  const onClickContinue = useCallback(() => {
+    setUserChoiceIndex(undefined);
+    if (stepIndex >= stepCount - 1) {
+      // TODO: handle continue pressed on last question;
+      onClose();
+      setStepIndex(0);
+    } else {
+      setStepIndex(stepIndex + 1);
+    }
+  }, [stepIndex, stepCount, setStepIndex, setUserChoiceIndex, onClose]);
+
+  const {
+    title: stepTitle,
+    choices,
+    answerExplanation,
+    correctAnswerExplanation,
+    CorrectAnswerIllustration,
+    correctAnswerTitle,
+    Illustration,
+    incorrectAnswerExplanation,
+    IncorrectAnswerIllustration,
+    incorrectAnswerTitle,
+  } = steps[stepIndex];
+
+  const userMadeAChoice = userChoiceIndex !== undefined;
+  const userChoice = userMadeAChoice ? choices[userChoiceIndex] : undefined;
+  const isCorrectChoice = userChoice ? userChoice.correct : undefined;
+
+  const radioName = `quizz-${title}-step-${stepIndex}`;
+
+  const AsideLeft = (
+    <Flex flexDirection="column" rowGap={12}>
+      <Text variant="h5">{stepTitle}</Text>
+      <Radio
+        name={`quizz-${title}-step-${stepIndex}`}
+        onChange={value => !userMadeAChoice && setUserChoiceIndex(value)}
+        currentValue={userChoiceIndex}
+        containerProps={{ flexDirection: "column", rowGap: 5 }}
+      >
+        {choices.map(({ label, correct }: QuizzChoice, index: number) => (
+          <RadioElement
+            key={`${radioName}-choice-${index}`}
+            label={label}
+            value={`${index}`}
+            variant={userMadeAChoice ? (correct ? "success" : "error") : "default"}
+          />
+        ))}
+      </Radio>
+    </Flex>
+  );
+
+  const rightSideIllustration =
+    (userMadeAChoice
+      ? isCorrectChoice
+        ? CorrectAnswerIllustration
+        : IncorrectAnswerIllustration
+      : Illustration) || Illustration;
+
+  const rightSideTitle = userMadeAChoice
+    ? userChoice?.specificAnswerTitle ||
+      (isCorrectChoice ? correctAnswerTitle : incorrectAnswerTitle)
+    : null;
+
+  const rightSideExplanation = userMadeAChoice
+    ? userChoice?.specificAnswerExplanation ||
+      (isCorrectChoice ? correctAnswerExplanation : incorrectAnswerExplanation) ||
+      answerExplanation
+    : null;
+
+  const rightSideBgColor = userMadeAChoice
+    ? isCorrectChoice
+      ? "success.c100"
+      : "error.c100"
+    : "primary.c60";
+
+  const AsideRight = (
+    <Answer
+      Illustration={rightSideIllustration}
+      illustrationSize={userMadeAChoice ? 176 : 280}
+      title={rightSideTitle}
+      description={rightSideExplanation}
+    />
+  );
+
+  const [width, height] = [816, 486];
```

Comment:
I think that these dimensions are not really used by the popin component since there are fixed max width/height values in the styled components theme:

```
popin: {
        min: {
          height: 158,
          width: 462,
        },
        max: {
          height: 522,
          width: 622,
        },
      },
```

Reply:
Yes indeed but actually I wanted to change this as there are bigger `popin` components in the [design](https://www.figma.com/file/q1bu6mRiwzIhBEXhszfsts/LLD-%2F-Onboarding?node-id=201%3A1672), i don't think we should go for a static max size like this (in px), maybe make it a percentage ...